ci/pipelines/node-runner.yml

```diff
@@ -0,0 +1,44 @@
+---
+resources:
+  - name: node-runner-src
+    type: git
+    icon: github
+    source:
+      uri: https://github.com/alphagov/pay-ci
+      branch: master
+      paths:
+        - ci/docker/node-runner/*
+
+  - name: node-runner
+    type: registry-image
+    icon: docker
+    source:
+      repository: govukpay/node-runner
+      username: ((docker-username))
+      password: ((docker-password))
+      tag: latest
```

Comment:
Are we able to give multiple tags on our initial push? So that as well as `latest` we have a tag in Dockerhub with the incremented release number (or date)?

Reply:
It doesn't look like we can. From https://github.com/concourse/registry-image-resource:
```
tag: Optional. Instead of monitoring semver tags, monitor a single tag for changes (based on digest).
```
My preference would be to specify an incremented release number/date over using "latest" if we have to choose.

Reply:
😿 OK. I guess we can iterate on that later if incrementing proves tricky right now - I'd rather have `latest` than no image there at all...

Reply:
I can try defining two registry-image resources.